lib/modules/manager/copier/artifacts.ts

```diff
@@ -1,4 +1,5 @@
 import { quote } from 'shlex';
+import { basename, dirname } from 'upath';
```

Comment:
```suggestion
import upath from 'upath';
```
otherwise we'll get esm import issues later